src/components/Modals/SetAclPermissionsModal.jsx

```diff
@@ -40,23 +39,24 @@ import useNotification from '../../hooks/useNotification';
 const SetAclPermissionsModal = ({ showModal, setShowModal, dataset }) => {
   const { session } = useSession();
   const { addNotification } = useNotification();
-
   const { podUrl, webId } = useContext(SignedInUserContext);
-
   const [permissionState, setPermissionState] = useState({
     webIdToSetPermsTo: '',
     permissionType: ''
   });
   const [processing, setProcessing] = useState(false);
-
   const { data } = useContactsList();
-  const contactListOptions = data?.map((contact) => ({
-    label: `${contact.person} ${contact.webId}`,
-    id: contact.webId
-  }));
+  const contactListOptions = data
+    ? data.map((contact) => ({
+        label: `${contact.person} ${contact.webId}`,
+        id: contact.webId
+      }))
+    : [];
```

Comment:
I think this is a good place to use `??` in place of a ternary since `?.` would help use handle the case when data is null or undefined and `??` would help us fall back to `[]` if it did end up as null or undefined.